dashboard/django/stats/views.py

```diff
@@ -282,3 +305,55 @@ def clusters(request):
     return render(request, 'stats/clusters.html', {
         'clusters' : clusters,
     })
+
+
+def clearSubscription(request, topic_name, subscription_name):
+    url = settings.SERVICE_URL + '/admin/v2/' + topic_name + '/subscription/' + subscription_name + '/skip_all'
+    requests.post(url)
+    return redirect('topic', topic_name=topic_name)
+
+def deleteSubscription(request, topic_name, subscription_name):
+    url = settings.SERVICE_URL + '/admin/v2/' + topic_name + '/subscription/' + subscription_name
```

Comment:
This option settings.SERVICE_URL needs to be included in the configuration file settings.

Reply:
Done